.github/workflows/interface_gen_script.sh

```diff
@@ -1,16 +1,16 @@
-#!/usr/bin/sh
+#!/usr/bin/bash
 
 # Generate Matlab interface using swig matlab
 mkdir build_matlab
-pushd build_matlab
+pushd build_matlab || exit
```

Comment:
what does the ||exit do exactly.  I would guess it just just forces the return result of this command to be 0?

Reply:
The default exit status is 0. The main thing is that if changing directories fails it stops running the rest of the script.